src/renderer/components/blocks/layout/IssuedCarbonYearlyChart.tsx

```diff
@@ -69,6 +70,12 @@ const IssuedCarbonYearlyChart = () => {
     );
   }
 
+  const unitStatusOptions = [
```

Comment:
due to how the `<Select>` component is written, this should be 

```
const unitStatusOptions = [ 
  capitalizeText(intl.formatMessage({ id: 'retired' })), 
  capitalizeText(intl.formatMessage({ id: 'held' })),
  capitalizeText(intl.formatMessage({ id: 'cancelled' }))
];
```

the label and value options are for specifying abbreviated or alternate values in the drop down vs the selected value. in this case when an option is selected, we want the text displayed as selected to be the same text that the user clicked in the drop down. the above array accomplishes this.

Reply:
The value is used during the API call, so if it were to be used as a translated value, it would not yield the correct results. Therefore, I think it is essential to have a value that accurately facilitates data retrieval using the API

Reply:
Ah I see, yea that would cause things to break. Let's leave it as is for now.